src/main/java/organice/logic/commands/ListCommand.java

```diff
@@ -1,24 +1,65 @@
 package organice.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static organice.logic.parser.CliSyntax.PREFIX_TYPE;
+import static organice.model.Model.PREDICATE_SHOW_ALL_DOCTORS;
+import static organice.model.Model.PREDICATE_SHOW_ALL_DONORS;
+import static organice.model.Model.PREDICATE_SHOW_ALL_PATIENTS;
 import static organice.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
 import organice.model.Model;
+import organice.model.person.Type;
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists all persons of the same type in the address book to the user.
  */
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Returns a list of persons of the stated type.\n"
+            + "Parameter:\n"
+            + PREFIX_TYPE + "PERSON TYPE ";
 
+    public static final String MESSAGE_SUCCESS = "Listed all persons of stated type";
```

Comment:
No need this i think, we have all the strings below just make use of them

Reply:
Removed this string.